posts/ambassador_alves_intro.html

```diff
@@ -0,0 +1,194 @@
+                                <p>Since I first started learning more about it, I have been hooked on quantum computing. While possibly over-hyped and over-promised by the media, the future impact this field of computing will likely have remains significant. I liken it to having the opportunity to get involved in learning about classical computing in the 1950s when computers consisted of mammoth, building-sized machines that could accomplish relatively little compared to what we now think of and associate with a computer. But the major difference between then and now is that in the 1950s, most people did not have access to those computers. Meanwhile today, anyone with a laptop and an internet connection can not only learn about quantum computing, but they can even <a href="https://quantum-computing.ibm.com/">run code on real quantum hardware for free</a>!</p>
+                                <h2 id="getting-involved-with-unitary-fund">Getting Involved with Unitary Fund</h2>
+                                <p>I participated in the <a href="https://qosf.org/mentorship_cohort_3/">third cohort</a> of the Quantum Open Source Foundation’s Quantum Mentorship Program, where my mentor Tom Lubowe suggested making a contribution to Mitiq as a good project for me. To apply for the mentorship program, QOSF required applicants to choose a challenge from a list of options and submit their unique solution along with their application. Consequently, I built a small, Python-based <a href="https://github.com/andre-a-alves/QOSF-Mentorship-Submission-Quantum-Simulator">quantum simulator</a>. Although I had already learned to code in C and Java, that simulator was my first Python project and the vehicle by while I learned that programming language, which I have since used for several open-source contributions as well as work, school, and personal projects.</p>
+                                <p>Once I started attending the weekly Mitiq meetings (every Friday at 17:00 UTC on <a href="http://discord.unitary.fund/">discord</a>) to understand what was going on, I was amazed at the open and welcoming community I found at Unitary Fund. Along with that sense of community, following along during the meetings helped me understand what were priority issues, and I decided to tackle creating decorators for PEC error correction. Decorators already existed for ZNE at the time, so this task was essentially making it so users could have the same or similar options for how to implement both techniques within their programs. In addition to that contribution, I made a few others that were mostly focused on organizing testing and improving documentation.</p>
```

Comment:
would be great to add a link to the PEC decorators
```suggestion
                                <p>Once I started attending the weekly Mitiq meetings (every Friday at 17:00 UTC on <a href="http://discord.unitary.fund/">discord</a>) to understand what was going on, I was amazed at the open and welcoming community I found at Unitary Fund. Along with that sense of community, following along during the meetings helped me understand what were priority issues, and I decided to tackle creating decorators for PEC, a quantum error mitigation technique available in Mitiq. Decorators already existed for ZNE at the time, so this task was essentially making it so users could have the same or similar options for how to implement both techniques within their programs. In addition to that contribution, I made a few others that were mostly focused on organizing testing and improving documentation.</p>
```

Reply:
Change incorporated, and I also added a link to PEC decorator as suggested